CMakeLists.txt

```diff
@@ -355,28 +355,41 @@ macro(nebula_add_test)
         nebula_test                 # prefix
         "DISABLED"                  # <options>
         "NAME"                      # <one_value_args>
-        "SOURCES;OBJECTS;LIBRARIES" # <multi_value_args>
+        "SOURCES;OBJECTS;LIBRARIES;FUZZER" # <multi_value_args>
```

Comment:
IMO, `FUZZER` is an option which carries no arguments, like `DISABLED` above. So you should place it in the _options_ list.

Reply:
Well, given a second thought, `FUZZER ON/OFF` is more expressive.